data-table/table.js

```diff
@@ -157,6 +157,42 @@ function addColumns(numberOfColumns, index) {
      */
 }
 
+/**
+ * This function takes the id of the list container for the column names
+ * It then adds to the list another space for another column at the bottom of the list
+ * @param config id of the table
```

Comment:
the config is the entire configuration now, not just the id of the table. Update this comment and the one for the next function to reflect that.

Reply:
I will make the appropriate comment changes. Thanks!